source/common/http/header_map_impl.h

```diff
@@ -520,6 +543,11 @@ class RequestTrailerMapImpl final : public TypedHeaderMapImpl<RequestTrailerMap>
     return std::unique_ptr<RequestTrailerMapImpl>(new (inlineHeadersSize())
                                                       RequestTrailerMapImpl());
   }
+  static std::unique_ptr<RequestTrailerMapImpl> create(const uint32_t max_headers_kb,
+                                                       const uint32_t max_headers_count) {
+    return std::unique_ptr<RequestTrailerMapImpl>(
```

Comment:
we should just `make_unique` here and elsewhere

https://abseil.io/tips/126

Reply:
What's the syntax to use make_unqiue and the inlineHeadersSize() ?

Reply:
Interesting, looks like `InlineStorage` has its own new operator. Also all other places in this file are using this style. I can take a closer look later.

It is fine to keep as it is.